workshop_app/models.py

```diff
@@ -200,26 +200,26 @@ def __str__(self):
         )
 
 
-class ProfileComments(models.Model):
+class WorkshopComment(models.Model):
```

Comment:
Please just change all the name references to Comment, since there is a foreign key we will know that the comments are for workshops.